docs/user/00-30-vpc-peering.md

```diff
@@ -1,5 +1,7 @@
 # VPC Peering
```

Comment:
Don't you need this note somewhere in this doc: VPC peering for Microsoft Azure is a feature available only for SAP-internal teams.? I think there should be one, but maybe I'm wrong ;)